### PR TITLE
Add comment to explain purpose of SULU_ADMIN_EMAIL environment variable

### PR DESCRIPTION
**Pull request by nnatter**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -36,6 +36,7 @@ DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.
 ###< doctrine/doctrine-bundle ###
 
 ###> sulu/sulu ###
+# Email address that is used as sender address for emails sent by Sulu (eg. password reset email)
```
Comment (danrot) [inline .env]: IMO this comment is misleading... The email address is only used for the example you've given, if the user which uses this functionality has not an email address...

Comment (nnatter) [reply .env]: how should we describe it? would prefer to have some comment here 🙂

Comment (danrot) [reply .env]: Well, is that the only use case we are using that email address? If yes, I would add that it is only used as a fallback in the above mentioned case. If I read it as it is, I would wonder as a user, why every password forgotten email is sent to the admin :see_no_evil:

Comment (alexander-schranz) [reply .env]: The form and community bundle does use the sulu_admin.email parameter as default value.

**Pull request merged**